### ISSUE TITLE
Fix accessibility issue, hint text has insufficient contrast to background in the "add another" blocks 

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#73305]
## Issue Description

Slack thread: https://dsva.slack.com/archives/C05MS96P6P2/p1704846613101069?thread_ts=1704484692.155699&cid=C05MS96P6P2

Accessibility issue on a few pages:

- https://staging.va.gov/pension/application/527EZ/financial/income-sources
- https://staging.va.gov/pension/application/527EZ/household/marital-status/spouse-marriages
- https://staging.va.gov/pension/application/527EZ/financial/medical-expenses/add

Hint text has insufficient contrast to background in the “add another” blocks.

<img width="1578" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNkd4QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6c05dae14a2facfd0cc4fee87e0a7bf05bd1d018/not-enough-contrast.png" alt="not-enough-contrast.png" />

## Acceptance Criteria
- [ ] The hint text found in list loops of the pages listed above will have enough color contrast to meet accessibility standards

**Comment by aplatt-coforma** [comment]
Contrast looks good on my monitor but not sure what it looked like before. 

<img width="885" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd2pSQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0a0d5a5e9c93b43bbeea1231da988bdf57d4dd76/Screenshot%202024-01-16%20at%205.43.11%E2%80%AFPM.png" alt="Screenshot 2024-01-16 at 5.43.11 PM.png" />
<img width="610" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd25SQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--bbffab8fc2e3782b854d3898869cb14820291aea/Screenshot%202024-01-16%20at%205.44.33%E2%80%AFPM.png" alt="Screenshot 2024-01-16 at 5.44.33 PM.png" />
<img width="717" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3JSQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ce217e0951da2bd13289a4471ec0dd0a8bcdc4e0/Screenshot%202024-01-16%20at%205.46.18%E2%80%AFPM.png" alt="Screenshot 2024-01-16 at 5.46.18 PM.png" />
